QSort/QSort/QSort.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace QSort
+{
+    public  class QSort<T> where T:IComparable
+    {
+        private IList<T> List;
```

Comment:
```suggestion
        private IList<T> list;
```
camelCase же. Если Вы и HwProj так пишете, мы все обречены :)